docs/content/en/docs/pipeline-stages/init.md

```diff
@@ -176,3 +178,14 @@ deploy:
     - leeroy-app/kubernetes/deployment.yaml
     - leeroy-web/kubernetes/deployment.yaml
 ```
+
+### Exit Codes
+
+When `skaffold init` fails to create a valid config, it exits with an code that depends on the error:
+
+| Exit Code | Error |
+| ---- | --- |
+| 101 | No build configuration could be found |
+| 102 | No k8s manifest could be found or generated |
+| 102 | A pre existing skaffold.yaml prevents the creation of the config |
+| 104 | Couldn't match builder with image names automatically |
```

Comment:
Does this apply to `--analyze`?  Or is it only for `--analyze`?

Reply:
It also applies to `--analyze` although `skaffold init --analyze` should fail less often. For example, it won't fail if there's no k8s manifest, I think